src/cli.c

```diff
@@ -1364,46 +1257,454 @@ int main(int argc, char **argv)
             goto end_program;
         }
 
-        if (strcmp(command, "download-file") == 0) {
-            char *bucket_id = argv[command_index + 1];
-            char *file_id = argv[command_index + 2];
-            char *path = argv[command_index + 3];
+        storj_api = malloc(sizeof(storj_api_t));
 
-            if (!bucket_id || !file_id || !path) {
-                printf("Missing arguments: <bucket-id> <file-id> <path>\n");
+        if (!storj_api) {
+            status = 1;
+            goto end_program;
+        }
+        memset(storj_api, 0x00, sizeof(*storj_api));
+
+        storj_api->env = env;
+
+        #ifdef debug_enable
```

Comment:
There is a logger that can be used that will enable/disable debug based on settings (e.g. using --debug), example usage:
```
state->log->debug(env->log_options, NULL /*handle pointer*/, "Some debug message: %i", some_integer);
````

Reply:
currently printf's that I left in the code is used to printout error conditions. Will look into adding log->debug usage